src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/EncryptDecrypt.cs

```diff
@@ -150,10 +152,8 @@ public void DecryptSavedAnswer_OaepSHA256()
 
             byte[] output;
 
-            using (RSA rsa = RSAFactory.Create())
+            using (RSA rsa = RSAFactory.Create(TestData.RSA2048Params))
             {
-                rsa.ImportParameters(TestData.RSA2048Params);
-
                 if (RSAFactory.SupportsSha2Oaep)
```

Comment:
I went through every call to RSAFactory.Create or RSA.Create to find out if they could be changed to pooled keys, or not.  All the ones that did ImportParameters as their first line are from .NET Core 1.0 (hi, former self!), before we had accelerators for that.  They generally got changed over so they a) didn't look like false positives and b) can suggest other key parameter values that are worth pooling as well-known keys.